test/utils_test.py

```diff
@@ -0,0 +1,49 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+from unittest import TestCase
+
+from pybutton.utils import is_webhook_authentic
+
+
+class UtilsTestCase(TestCase):
+
+    def test_is_webhook_authentic(self):
+        signature = (
+            '79a3a5291c94340ff0058a631906375'
+            + '768d706357ee86826c3c692e6b9aa6817'
```

Comment:
Not a big deal, but FYI python will automatically concat strings across new lines so you don't need the `+`